styleguide/components/hero/animations.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CSSTransition from 'react-transition-group/CSSTransition';
+import astroStyles from './astronauts.css';
+import Stars from './stars';
+
+function getTransition(klass, appearKlass, appearActiveKlass) {
```

Comment:
INFO: I would just avoid defining the variables below in favor of just using the parameters defined here.